### PR TITLE
Add v2_image_client.py example

### PR DESCRIPTION
**Pull request by CoderHam**
Replicate image_client.py with V2 protocol
Follow up in upcoming PRs: streaming, folders as input instead of image, async

### CODE REVIEW

**Comment by deadeyegoodwin** [comment]
@CoderHam Do you have a CI for this?

**Comment by CoderHam** [comment]
> @CoderHam Do you have a CI for this?

Started one

File: src/clients/python/experimental_api_v2/examples/CMakeLists.txt [hunk 1]
```diff
@@ -57,3 +57,9 @@ if(${TRTIS_ENABLE_GRPC_V2})
     DESTINATION python
   )
 endif() # TRTIS_ENABLE_GRPC_V2
+
+install(
```
Comment (deadeyegoodwin) [inline src/clients/python/experimental_api_v2/examples/CMakeLists.txt:61]: Should only install if V2 GRPC || V2 HTTP

**Comment by CoderHam** [comment]
> Does this need a rebase? I don't see the class clients being delete here.

Yes. Rebased now.

**Pull request merged**